src/app/projects/components/create-new-attribute/create-new-attribute.component.ts

```diff
@@ -424,4 +436,9 @@ export class CreateNewAttributeComponent implements OnInit, OnDestroy {
     const statement = "from knowledge import " + pythonVariable;
     navigator.clipboard.writeText(statement);
   }
+
+  updateVisibilityAttributes(value: string) {
+    this.currentAttribute.visibility = value;
+    this.saveAttribute(this.project.id);
```

Comment:
will this run two queries

Reply:
It runs the mutation for updatingAttribute and after that the queries getAttributesByProjectId and getAttributeByAttributeId